05_Class_setup.Rmd

```diff
@@ -2,34 +2,414 @@
 
 # Class Setup
 
+## Overview of Class Setup
+
+This section will show you how to organize your class to make it easier to administer access to your content. You will need to have a list of who will be taking your class, such as a course roster or sign-up list, as well as a list of additional instructors or teaching assistants. You can make changes later, so the list of students need not be final.
+
 ## Identify Funding
 
 ### NHGRI STRIDES {#strides-funding}
 
 Through the [STRIDES](https://datascience.nih.gov/strides) program, NHGRI makes funding available to new users of the AnVIL, in collaboration with the [NIH Office of Data Science Strategy](https://datascience.nih.gov/about/odss). NHGRI currently supports the [AnVIL Cloud Credits Program (AC2)](https://anvilproject.org/news/2021/04/12/announcing-anvil-cloud-cost-credits-program) and will have a second round of funding available in fall of 2021.
 
-## Set Up Billing Project
+## Collect Google IDs
+
+AnVIL IDs are based on [Google accounts](account-setup.html#google-account).
+
+Students -- Contact students/participants to get their AnVIL IDs. These should be Gmail addresses or emails with GSuite capabilities. You can link students to [Student Account Setup](student-account-setup.html) for instructions on what they should do.
+
+Co-instructors -- If you will be working with other instructors, such as co-instructors or teaching assistants, you will need to collect their IDs as well.
+
+## Set Up Billing Projects
+
+Billing Projects in Terra help organize your compute costs. We suggest creating two different billing projects under the appropriate Billing Account that you created on `cloud.google.com`: one for students and one for co-instructors.
+
```

Comment:
This is great, and I agree it's important to have a reminder. Is something like this better suited for the "Account Setup" section, and we can link back to it here?

Reply:
I think that makes sense, to have the long version when they make their first Billing Account, and then link back to it here.  For context, I'm imagining the teacher coming back to teach it for the second time, going straight to the Class Setup, and trying to remember how to set things up.

I think here it would be useful to have a reminder that "Billing Accounts" are the things on Google, and "Billing Projects" are the things on Terra (just because the names are so similar and confusing), and then link back to the longer explanation if they need more info.

So maybe something more like:

```suggestion
Reminder:

- Google **Billing Accounts** are managed on Google Cloud Platform and are used for organizing *funding sources* (e.g. credit cards, cloud credits).
- Terra **Billing Projects** are managed through Terra, and allow you to associate your Terra activity with the correct Google Billing Account.

For a more detailed explanation, please see the chapter on Account Setup.

We suggest creating two different Terra Billing Projects under the appropriate Billing Account that you created on `cloud.google.com`: one for students and one for co-instructors. The instructions below will walk you through how to set this up.

```

Reply:
Sounds great - thanks for thinking so much about this! I'll make an issue to make sure this is referred to explicitly earlier in the guide. We'll want to update the links when the account stuff is filled in.